src/renderer/mod.rs

```diff
@@ -745,6 +771,13 @@ impl<'a> Renderer<'a> {
 
         for n in body {
             output.push_str(&self.render_node(n)?);
+            if let Some(for_loop) = self.current_for_loop() {
+                match for_loop.state {
+                    ForLoopState::Continue
+                    | ForLoopState::Break => break,
```

Comment:
why is it breaking on continue?

Reply:
It breaks on continue to stop processing the remaining statements in the iteration. The next iteration of the for loop in the template still executes